google/cloud/alloydb/connector/instance.py

```diff
@@ -83,6 +82,25 @@ def __init__(
         self._current: asyncio.Task = self._schedule_refresh(0)
         self._next: asyncio.Task = self._current
 
+    def _parse_instance_uri(self, instance_uri: str) -> Tuple[str, str, str, str, str]:
```

Comment:
let's move this function to the top of the file and not have it as a standalone function instead of a class method.
```suggestion
    def _parse_instance_uri(instance_uri: str) -> Tuple[str, str, str, str]:
```

Reply:
Removed the extra return type for str from the Type Check return. Function is moved at the top of the file.